src/hooks/notice/detail/useDeleteNoticeMutation.hook.ts

```diff
@@ -0,0 +1,15 @@
+import { goto } from '$app/navigation';
+import { ROUTER } from '@/constants';
+import deleteNotice from '@/services/notice/detail/deleteNotice';
+import { createMutation } from '@tanstack/svelte-query';
+
+const useDeleteNoticeMutation = () => {
+	return createMutation({
+		mutationFn: deleteNotice,
+		onSuccess: () => {
+			goto(ROUTER.NOTICE);
+		}
```

Comment:
한 줄로 표현하는 방식은 어떻게 생각하시나요?
추후에도 답변해주시는 의견대로 코드를 작성할 때도 맞춰보겠습니다!
```suggestion
onSuccess: () => goto(ROUTER.NOTICE);
```